src/helm/common/object_spec.py

```diff
@@ -17,16 +17,24 @@ def __hash__(self):
         return hash((self.class_name, tuple((k, self.args[k]) for k in sorted(self.args.keys()))))
 
 
-def create_object(spec: ObjectSpec, additional_args: Optional[Dict[str, Any]] = None):
-    """Create the actual object given the `spec`."""
-    # TODO: Refactor other places that use this pattern.
-    components = spec.class_name.split(".")
+def get_class_by_name(full_class_name: str) -> Type[Any]:
+    components = full_class_name.split(".")
     class_name = components[-1]
     module_name = ".".join(components[:-1])
-    cls = getattr(importlib.import_module(module_name), class_name)
+    return getattr(importlib.import_module(module_name), class_name)
+
+
+def create_object(spec: ObjectSpec, additional_args: Optional[Dict[str, Any]] = None):
+    """Create the actual object given the `spec`."""
```

Comment:
Elaborate on documentation of `additional_args`

Reply:
Hmm...I think it might be cleaner / more modular to have a function that takes a spec and updates the spec with additional_args, so that the `create_object` function can stay the same.